docs(k8s): update ssh documentation (#2405)
scaleway/docs-content: containers/kubernetes/how-to/enable-disable-ssh.mdx

---
meta:
  title: How to enable or disable SSH ports on Kubernetes Kapsule cluster nodes
  description: This page explains how to enable or disable SSH ports on Kubernetes Kapsule cluster nodes
content:
  h1: How to enable or disable SSH ports on Kubernetes Kapsule cluster nodes
  paragraph: This page explains how to enable or disable SSH ports on Kubernetes Kapsule cluster nodes
tags: kubernetes kapsule-cluster
dates:
  validation: 2023-11-14
  posted: 2023-05-02
categories:
  - containers
---

Kubernetes Kapsule clusters are provisioned with a default security group as standard. Clusters deployed prior to May 2023 permit inbound traffic to the nodes.<br /><br />
Since May 2023, each Kapsule cluster comes with a default security group implementing an inbound `DROP ALL` policy, effectively blocking all incoming connections by default.<br /><br />

> To activate the preinstalled SSH server for node access within the cluster, it is necessary to manually [configure a custom rule](/compute/instances/how-to/use-security-groups/#how-to-edit-a-security-group) in the cluster's security group to permit inbound traffic on TCP port 22.

Reviewer: ```suggestion
To activate the preinstalled SSH server for node access within the cluster, it is necessary to manually [configure a custom rule](/compute/instances/how-to/use-security-groups/#how-to-edit-a-security-group) in the cluster's security group to allow inbound traffic on TCP port 22.
```

  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
  - [This feature is available on the nodes of your cluster](#how-to-know-if-this-feature-is-available-on-the-nodes-of-your-cluster)
</Message>

> 1. Click **Instances** in the side menu, followed by **Security groups** to view the [security group](https://console.scaleway.com/instance/security-groups) section in your Scaleway console.

Reviewer: ```suggestion
1. Click **Instances** in the side menu, then click **Security groups** to view the [security group](https://console.scaleway.com/instance/security-groups) section in your Scaleway console.
```

2. Select "more" <Icon name="more"/> adjacent to the security group you intend to configure and choose **More info**. 
    <Message type="tip">

>       Alternatively, direct selection of the security group's name is possible. The security group's overview displays.

Reviewer: ```suggestion
      Alternatively, you can also select the security group's name. The security group's overview displays.
```

    </Message>
3. Access the **Rules** tab to view the security group rules.
4. In the **Rules** section of the security group, select "edit" <Icon name="edit" />.
5. Configure a new custom inbound rule:
    * Rule: accept
    * Protocol: TCP
    * Port: 22 (standard SSH port)

>     * IP range: Specify a CIDR IP range authorized for connection to your Instance. As an option, select **All IPs** to permit connections from any IP. It is recommended to resctrict access to your rule to a limited IP range for maximum security.

Reviewer: ```suggestion
    * IP range: Specify a CIDR IP range authorized for connection to your Instance. As an option, select **All IPs** to allow connections from any IP. It is recommended to restrict access to your rule to a limited IP range for maximum security.
```
